src/webpack-isomorphic-parser.js

```diff
@@ -19,7 +19,7 @@ const iconsFileRE = /icons-[a-f0-9]+\.svg/i;
  */
 const iconsReplaceRE = new RegExp(FILENAME_REPLACE_STRING, 'g');
 
-module.exports = (module, options) => {
+module.exports = (module, options = { path: '/static' }) => {
```

Comment:
So, it seems like you're not actually using `options.path` anywhere? Also, this function is only ever called by `webpack-isomorphic-tools`, so I don't _think_ a default param makes sense here. The passed-in `publicPath` option would have to be available to the parser somehow, though. Maybe with a higher-order function?

```javascript
module.exports = ({ publicPath }) => (module, options) => {
  if (publicPath == null) {
    publicPath = options.assets_base_url;
  }
  ...
```

something like that?